hearth/lib/hearth/middleware/retry.rb

```diff
@@ -73,20 +73,24 @@ def call(input, context)
           output.error = interceptor_error if interceptor_error
 
           if (error = output.error)
+            context.logger.debug('[Middleware::Retry] Request errored')
```

Comment:
should we have info about the error?

Reply:
Yeah, if it's not raised, we should probably log details.